build/android-cpu_features.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+if [[ -z ${ANDROID_NDK_ROOT} ]]; then
+    echo -e "(*) ANDROID_NDK_ROOT not defined\n"
+    exit 1
+fi
+
+if [[ -z ${ARCH} ]]; then
+    echo -e "(*) ARCH not defined\n"
+    exit 1
+fi
+
+if [[ -z ${API} ]]; then
+    echo -e "(*) API not defined\n"
+    exit 1
+fi
+
+# ENABLE COMMON FUNCTIONS
+. ${BASEDIR}/build/android-common.sh
+
+# PREPARE PATHS & DEFINE ${INSTALL_PKG_CONFIG_DIR}
+LIB_NAME="cpu_features"
+set_toolchain_clang_paths ${LIB_NAME}
+
+PREFIX=${BASEDIR}/prebuilt/android-$(get_target_build)/${LIB_NAME}
+
+android_cmake() {
```

Comment:
Using Android's cmake makes sense. What do you think about defining it globally inside `set_toolchain_clang_paths`? So, other libraries can use it too.

Reply:
Definitely, this can be moved to `android-common.sh`.

Reply:
What if I put a new 'function' **android_cmake()** into `android-common.sh`? 

This will handle running `cmake` with NDK toolchain and necessary arguments. You still can add more arguments to customize the build, e.g. `BUILD_SHARED_LIBS=ON`.

This same function can be used for all libraries that currently build with `cmake`, and hopefully, also help to switch other libraries, ones that support `cmake`, to avoid the slow `./configure`.